AltBeacon.Library/src/AltBeacon/Beacon/BeaconParser.cs

```diff
@@ -416,8 +417,7 @@ public BeaconParser SetBeaconLayout(string beaconLayout)
 
                 if (!found)
                 {
-                    ////TODO LogManager
-                    ////LogManager.d(Tag, "cannot parse term " + term);
+                    Logger.Debug("cannot parse term " + term);
```

Comment:
All of the debug logs inside the parser probably need to be wrapped in an if statement that checks to see if debug is enabled before constructing the string.  It is really important to do that here because this code will run hundreds of times per second when lots of bluetooth devices are around.